### DISCUSSION TITLE
Add Stroeer SSP as adopter in README.md please

### DISCUSSION THREAD

**Discussion by kroening-stroeer** [openooh/venue-taxonomy#96] [Q&A] [answered]
Add Stroeer SSP as adopter in README.md please

Platform | Status | Version | Notes
Stroeer SSP| Production | 1.0 | Supports OpenOOH v1.0.0 as of 10/01/21

Thank you
Sebastian

Director Program Management Public Video
Ströer

**Comment by kroening-stroeer** [comment]
@all Huhu !!

**Reply by kroening-stroeer** [reply]
Anyone reading this from the working group?? @jayshao jayshao

**Reply by kroening-stroeer** [reply]
@christiancollins11 reading here?

**Reply by jayshao** [reply]
@kroening-stroeer sorry - have been watching PRs more closely than discussion. Check to make sure I got it right?

https://github.com/openooh/venue-taxonomy/blob/main/README.md

**Comment by jayshao** [comment, marked as answer]
https://github.com/openooh/venue-taxonomy/blob/main/README.md

**Comment by kroening-stroeer** [comment]
Fantastic! Many thanks!

Sebastian

—
Sebastian Kröning
Director Program Management Public Video
***@***.***
+49-175-2621777
________________________________
Von: Jason Shao ***@***.***>
Gesendet: Dienstag, August 16, 2022 19:19
An: openooh/venue-taxonomy ***@***.***>
Cc: Kröning Sebastian ***@***.***>; Mention ***@***.***>
Betreff: Re: [openooh/venue-taxonomy] Add Stroeer SSP as adopter in README.md please (Discussion #96)


@kroening-stroeer<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkroening-stroeer&data=05%7C01%7Csebastian.kroening%40stroeer.de%7C384dc52b7170420101a408da7fab799f%7C8fbdeb20138e4339ba6b5fea1e6efb54%7C0%7C0%7C637962671708833087%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sCURmf3rnvmL8vRrd%2F927uztsHOIzOGnQPECdFCaiu8%3D&reserved=0> sorry - have been watching PRs more closely than discussion. Check to make sure I got it right?

https://github.com/openooh/venue-taxonomy/blob/main/README.md<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fopenooh%2Fvenue-taxonomy%2Fblob%2Fmain%2FREADME.md&data=05%7C01%7Csebastian.kroening%40stroeer.de%7C384dc52b7170420101a408da7fab799f%7C8fbdeb20138e4339ba6b5fea1e6efb54%7C0%7C0%7C637962671708833087%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=XhL30Cg3fupVucQuR5TxQBDRujcm3B8%2FMKHXVuVNVIs%3D&reserved=0>
